pxr/imaging/hdSt/testenv/testHdStMaterialXShaderGen.cpp

```diff
@@ -160,7 +160,7 @@ int main(int argc, char *argv[])
             if (npos != std::string::npos) {
                 const std::string mx = textureMap.substr(0, npos);
                 const std::string hd = textureMap.substr(npos + 1);
-                mxHdInfo.textureMap[mx] = hd;
+                mxHdInfo.mxHdTextureMap[mx] = hd;
```

Comment:
In actual production Hydra code, the keys and the values in this map are always identical, i.e. it could have been a `std::set`. That's because MaterialX and Hydra names are anonymized identically by the topology cache mechanism https://github.com/PixarAnimationStudios/OpenUSD/pull/3073

These tests are the only place where the keys and the values differ, because the tests don't apply the topo cache optimization.